Core/Utilities/MySQL.py

```diff
@@ -1321,8 +1322,10 @@ def getFields( self, tableName, outFields = None,
     except Exception as x:
       return S_ERROR( DErrno.EMYSQL, x )
 
-    return self._query( 'SELECT %s FROM %s %s' %
-                        ( quotedOutFields, table, condition ), conn, debug = True )
+    if distinct:
+      return self._query( 'SELECT DISTINCT Name FROM %s %s' % ( table, condition ), conn, debug = True )
+    else:
+      return self._query( 'SELECT %s FROM %s %s' % ( quotedOutFields, table, condition ), conn, debug = True )
```

Comment:
This is a generic class used by all the mysql DBs in DIRAC, and I don't think it is a good idea to solve a specific case like the one of this PR by changing this class. Also, "Name" is only meaningful for a small subset of tables.